src/request_collector/cli.py

```diff
@@ -109,7 +109,7 @@ def main(
         monitoring_channel,
     )
 
-    database = StateDBSqlite(state_db)
+    database = StateDBSqlite(state_db + log_level)
```

Comment:
what is the ` log_level` for?

Reply:
Just left over from debugging. I thought maybe the db was still locked when the test hang. That must be removed, of course.